File: tests/openmp_examples/affinity/affinity_display.1.c
#pragma omp begin declare variant match(device={kind(host)})
#pragma omp end declare variant
#pragma omp begin declare variant match(device={kind(nohost)})
#pragma omp end declare variant
#pragma omp parallel num_threads(omp_get_num_procs())
#pragma omp parallel num_threads( omp_get_num_procs() )
#pragma omp parallel num_threads( omp_get_num_procs()/2 )

File: tests/openmp_examples/affinity/affinity_display.1.f90
!$omp    parallel num_threads(omp_get_num_procs())
!$omp    end parallel
!$omp    parallel num_threads( omp_get_num_procs() )
!$omp    end parallel
!$omp    parallel num_threads( omp_get_num_procs()/2 )
!$omp    end parallel
